kie-server-parent/kie-server-services/kie-server-services-jbpm/src/main/java/org/kie/server/services/jbpm/RuntimeDataServiceBase.java

```diff
@@ -189,7 +190,19 @@ public ProcessInstanceList getProcessInstancesByDeploymentId(String containerId,
         return processInstanceList;
     }
 
+    public CountDefinition countProcessInstancesByDeploymentId(String containerId, List<Integer> status) {
+        if (status == null || status.isEmpty()) {
+            status = new ArrayList<>();
+            status.add(ProcessInstance.STATE_ACTIVE);
+        }
+        logger.debug("About to search for process instance belonging to container '{}' with page {} and page size {}", containerId);
```

Comment:
wrong number of parameters expected by the log
```suggestion
        logger.debug("About to search for process instance belonging to container '{}'", containerId);
```